cg/meta/rsync/rsync_api.py

```diff
@@ -201,13 +201,19 @@ def slurm_rsync_single_case(
             sample_files_present=sample_files_present,
             case_files_present=case_files_present,
         )
+        existing_folder_list: List[str] = [
+            folder
+            for folder in folder_list
+            if (source_and_destination_paths["delivery_source_path"].joinpath(folder).exists())
+        ]
```

Comment:
Should this be part of `verify_folder_existance`?

Reply:
I think I was on the fence about creating a `verify_folder_existance` function. Do you think it would improve the readability?